index.html

```diff
@@ -0,0 +1,296 @@
+            <li><a href="linkedin.com" class="fa-brands fa-linkedin-in"></a></li>
+            <li><a href="medium.com" class="fa-brands fa-medium"></a></li>
+            <li><a href="angellist.com" class="fa-brands fa-angellist"></a></li>
+          </ul>
+        </div>
+      </div>
+      <div id="action">
+        <button class="normalbutnaboutt"> Get my resume</button>
+      </div>
+      
+    </div>
+
+    <div class="skillist">
+      <div class="dropdowncontainer">
+        <div id="dropdownheader">
+          <h3 class="droptext">Languages</h3>
+          <i class="dropdownhamburger">
+            <i class="fa-solid fa-angle-down"></i>
+          </i>
+        </div>
+        <div class="dropdown">
+          <div id="javadropdown">
+            <img src="./asset/About/javaScript.svg" alt="" class="image">
+            <h4 class="droptxt">Java</h4>
+          </div>
+          <div id="javadropdown">
+            <img src="./asset/About/Html.svg" alt="" class="image">
+            <h4 class="droptxt">HTML</h4>
+          </div>
+          <div id="javadropdown">
+            <img src="./asset/About/Css.svg" alt="" class="image">
+            <h4 class="droptxt">CSS</h4>
+          </div>        
+        </div>         
+      </div>
+      
+      <div class="dropdowncontainer">
+        <div id="dropdownheader">
+          <h3 class="droptext">Frameworks</h3>
+          <i class="dropdownhamburger">
+            <i class="fa-solid fa-angle-right"></i>
+          </i>
+        </div>
+      </div>
+      <hr class="hr"></hr>
+      <div class="dropdowncontainer">
+        <div id="dropdownheader">
+          <h3 class="droptext">Skills</h3>
+          <i class="dropdownhamburger">
+            <i class="fa-solid fa-angle-right"></i>
+          </i>
+        </div>
+      </div>
+      <hr class="hr"></hr>
+      
+    </div>
+  </section>
+
+  <section class="contactme">
+    <div class="contactblock">
+    
+        <h1 class="contacttxt">Contact me</h1>
+      
+      
+      <div class="contactbodytext">
+        <p 
+          class="body-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it 
+        </p>
+      </div>
+      
+      <form class="	form" action="https://formspree.io/f/xnqrezql" method="post" >
+        <input type="text" id="name" name="user_name" placeholder="FullName"  required maxlength="30"/>
+        <input type="email" id="mail" name="user_email" placeholder="Email Address"  pattern="/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+.[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-])*$/" required />
+       
+        <textarea id="textarea" cols="30" rows="10" placeholder="Write your message here" name="user_message" required maxlength="500"></textarea>
+       
+        <div id="action">
+          <button type="submit" class="normalbuttnabout" onclick=""> Get in Touch</button>
+        </div>
```

Comment:
- [ ] Kindly make sure the Get in touch button is adjusted as per the Figma design to avoid any bug :recycle: 
![Screenshot from 2023-03-04 14-21-55](https://user-images.githubusercontent.com/97834160/222900913-4b3e9074-93a9-47ef-a706-3fdff5970fc8.png)